draft-ietf-rtcweb-jsep.xml

```diff
@@ -1718,11 +1745,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           or not depends on the bundle policy.
           </t>
 
-          <t>The next step is to generate session-level lip sync groups as
-          defined in <xref target="RFC5888"/>, Section 7.  For each MediaStream
-          with more than one MediaStreamTrack, a group of type "LS" MUST be
-          added that contains the mid values for each MediaStreamTrack in that
-          MediaStream. </t>
+          <t>The next step is to generate session-level lip sync
+          groups as defined in <xref target="RFC5888"/>, Section
+          7. For each MediaStream referenced by more than one
+          RtpTransceiver, a group of type "LS" MUST be added that
```

Comment:
Both addTrack and addTransceiver take a list of MediaStreams, and that's how they are referenced.   Are you simply suggesting that we say "MediaStream referenced by more than one RtpTransceiver (via the addTrack and addTransceiver methods)"?